.github/workflows/test.yml

```diff
@@ -355,7 +415,7 @@ jobs:
     timeout-minutes: 10
```

Comment:
Just noticed we already have `client-types-napi`

Is there any type difference between napi or not?

Reply:
No, there shouldn't be! So we should be able to remove that

Reply:
Removed